src/status_im2/contexts/chat/composer/effects.cljs

```diff
@@ -186,6 +156,56 @@
        (rf/dispatch [:mention/on-change-text input-text])))
    [input-text]))
 
+(defn link-previews
```

Comment:
is this a new functionality? could you elaborate please

Reply:
Yes. Explained in comment below.